bundles/net.tourbook.cloud/src/net/tourbook/cloud/dropbox/PrefPageDropbox.java

```diff
@@ -226,6 +226,16 @@ public void init(final IWorkbench workbench) {
       //Not needed
    }
 
+   @Override
+   public boolean okToLeave() {
+
+      if (_server != null) {
+         _server.stopCallBackServer();
+      }
+
+      return super.okToLeave();
+   }
```

Comment:
I had still a case when the port was blocked but it occured after I clicked many times on the authorized button and using an old instance of the opened web page, I think this is OK. 

When this tool is used as a normal user then it works as expected.

Reply:
I knew that case too but a user shouldn't click several times on authorize. But sometimes users make mistake....